string source_robot
swarmus_ros_simulation/InterLocalization_msg[] otherRobots